scripts/build_test.py

```diff
@@ -724,22 +724,26 @@ def mock_require_compiled_js_dir_to_be_valid():
                     os.path.exists(os.path.dirname(MOCK_COMPILED_JS_DIR)))
 
     def test_generate_app_yaml(self):
-        temp_file = tempfile.NamedTemporaryFile()
-        temp_file.name = 'app_dev.yaml'
-        with open('app_dev.yaml', 'w') as tmp:
-            tmp.write('Some content in app_dev.yaml')
+        app_dev_yaml_file_swap = self.swap(
+            build, 'APP_DEV_YAML_FILE', 'core/tests/data/mock_app_dev.yaml')
```

Comment:
Define the two 'core/tests/...' as local named constants.  You use them multiple times and so they should not be hardcoded strings.

Reply:
Done